docs/source/overview/profiles_and_parameters.rst

```diff
@@ -497,7 +497,7 @@ For each Parameter with this Config File value on the same :ref:`Profile <profil
 
 .. _tm-related-cache-server-params:
 
-rascal.properties
+traffic_monitor.properties
 '''''''''''''''''
```

Comment:
This title underline is now too short, needs to be expanded to match the new title length